### PR TITLE
Add youtube video and examples

### PR DESCRIPTION
**Pull request by bvdmitri**

### CODE REVIEW

File: content/_index.md [hunk 1]
```diff
@@ -118,14 +125,15 @@ screenshot:
 
 ##################### Call to action #####################
 call_to_action:
-  enable : false
-  title : "Ready to get started?"
-  image : "images/rxinfer.svg"
-  content : "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Consequat tristique eget amet, tempus eu at consecttur."
+  enable : true
+  title : "Excited?"
```
Comment (bartvanerp) [inline content/_index.md:129]: Are you Excited?

Comment (bartvanerp) [reply content/_index.md:129]: or "Watch our talk!"

**Review by bartvanerp — APPROVED** [review]
Looks cool! I like the examples and video. I personally think it would be nicer to have the video first and then the examples, as there are multiple examples, so it takes some time before the video pops up.

I have some concerns regarding the examples, as I feel like Chrome is struggling to load and play them.  I don't know how bad this is on Safari, but I can image that mobile devices won't like it (as won't the hosting provider). In total the gifs are around 18MB, which is quite a lot. Maybe we could add them as autoplaying Youtube frames. Another solution would be to only display a single demo at a time, but with a menu with thumbnails to select the demo from (similarly as Bert proposed for the LD website). I have drawn some ideas on the whiteboard in the office

File: content/_index.md (hunk 1 above)
Comment (bartvanerp) [reply content/_index.md:129]: or "Want to know more?"

**Comment by bvdmitri** [comment]
Oh man, 18mb, I didn't even check. I will look into other solutions indeed

**Comment by bauglir** [comment]
> I will look into other solutions indeed

I'd try .webm, it's widely supported (in evergreen browsers at least) and produces small videos. .mov is a QuickTime format and proprietary and not as well supported (at least traditionally).

**Comment by bvdmitri** [comment]
I converted the videos to `.webm`, but it still is about 10mb. But as far as I understand those are not loading until the user scrolls till them and once they are loaded they are in cache

**Pull request merged**